README.md

```diff
@@ -40,56 +40,194 @@ exports.passport = {
 };
 ```
 
-### Using github and twitter strategy
+### Using Github and Twitter strategy
 
 ```js
-// app.js
+// config/config.default.js
+exports.passportGithub = {
+  clientID: 'my clientID',
+  clientSecret: 'my clientSecret',
+};
+
+exports.passportTwitter: {
+  consumerKey: 'my consumerKey',
+  consumerSecret: 'my consumerSecret',
+};
+```
+
+### Authenticate Requests
 
-const GithubStrategy = require('passport-github2').Strategy;
-const TwitterStrategy = require('passport-twitter').Strategy;
+Use `app.passport.authenticate()`, specifying the `'github'` and `'twitter'` strategy, to authenticate requests.
 
+```js
+// app/router.js
 module.exports = app => {
-  const githubStrategy = new GithubStrategy({
-    consumerKey: app.config.github.consumerKey,
-    consumerSecret: app.config.github.consumerSecret,
-    // authURL: '/passport/auth/github',
-    // callbackURL: '/passport/auth/github/callback',
-    // scope: [ 'user:email' ],
-  }, function* (accessToken, refreshToken, profile, done) {
-    const user = yield User.findOrCreate(...);
-    // user must contains `id` property
-    return user;
-  });
+  app.get('/', 'home.index');
 
-  const twitterStrategy = new TwitterStrategy({
-    consumerKey: app.config.twitter.consumerKey,
-    consumerSecret: app.config.twitter.consumerSecret,
-    // authURL: '/passport/auth/twitter',
-    // callbackURL: '/passport/auth/twitter/callback',
-  }, function* (token, tokenSecret, profile, done) {
-    const user = yield User.findOrCreate(...);
-    // user must contains `id` property
-    return user;
-  });
+  // authenticates routers
+  const github = app.passport.authenticate('github');
+  app.get('/passport/github', github);
```

Comment:
### Authenticate Requests

Use `app.passport.auth(provider, loginUrl)`, specifying the `'github'` and `'twitter'` strategy, to authenticate requests.

```js
// app/router.js
module.exports = app => {
  app.get('/', 'home.index');

  // authenticates routers
  app.passport.auth('github', '/passport/github');
  // this is a passport router helper, it's equal to below codes
  //
  // const github = app.passport.authenticate('github');
  // app.get('/passport/github', github);
  // app.get('/passport/github/callback', github);

  app.passport.auth('twitter', '/passport/twitter');
};
```